src/sentry/services/hybrid_cloud/notifications/impl.py

```diff
@@ -82,14 +87,30 @@ def get_settings_for_users(
         types: List[NotificationSettingTypes],
         users: List[RpcUser],
         value: NotificationSettingOptionValues,
+        organization: Organization | None = None,
     ) -> List[RpcNotificationSetting]:
-        settings = NotificationSetting.objects.filter(
-            user_id__in=[u.id for u in users],
-            type__in=types,
-            value=value.value,
-            scope_type=NotificationScopeType.USER.value,
+        if not organization or not features.has(
```

Comment:
@snigdhas We probably should think about how feature flagging works if there isn't an org

Reply:
yeah - not sure about this. There might be a point upstream where we have access to the org? don't know if that's always the case